lib/Elastica/Suggest/Phrase.php

```diff
@@ -171,13 +171,14 @@ public function toArray()
             $generator = $array[$baseName]['candidate_generator'];
             unset($array[$baseName]['candidate_generator']);
 
+            /**
+             * TODO: if covers - try use `key($generator)` and `current($generator)`
```

Comment:
I remove `redundant` code - and allright.
But also want change to code in `todo`:
```php
$array[$baseName][key($generator)][] = current($generator);
```